src/test/resources/net/sf/jabref/importer/fileformat/InspecSilverPlatterImporterTest.txt

```diff
@@ -1,30 +0,0 @@
-Record INSPEC
```

Comment:
It is used in testIsNotRecognizedFormat. So there is no specific test for the fields. Does this answer your question?